namespaces/cloud-platform-live-0.k8s.integration.dsd.io/laa-fala-staging/resources/ecr.tf

```diff
@@ -6,16 +6,36 @@ provider "aws" {
   region = "eu-west-1"
 }
 
-module "ecr-repo" {
+module "ecr-repo-laa-fala-webapp" {
```

Comment:
Renaming the resource will have the effect of the previous one being deleted and replaced by a new one